CONTRIBUTING.md

```diff
@@ -1,47 +1,117 @@
-  fixes should be sent to the next minor release update (e.g. 17.1.0 -> 17.1.1)
-
-## Ways To Get Started
-
-If you're looking for ideas for a way to contribute but don't know where to get
-started, some ideas to get you started:
-
-* You can browse some of our public [Issues](https://github.com/aces/Loris/issues)
+You should base your pull requests on one of the following branches
+depending on the kind of change you are making:
+
+#### Bug Fixes
+   - Branch: `bugfix` - Label:
+   ![](https://via.placeholder.com/15/cc9966/000000?text=+) **[branch]
+   bugfix** - Content: Generally these changes do not require SQL scripts
+   and are concise with the sole objective to correct a single problem
+   in the code.
+
+#### Minor Changes and Small Features
+   - Branch: `minor` - Label:
+   ![](https://via.placeholder.com/15/996633/000000?text=+) **[branch]
+   minor** - Content: Features affecting self-contained components such
+   as modules. Additions to Libraries, API, or modules that do not change
+   any function signatures.
+
+#### Major Changes, Non Backwards-Compatible Changes and Large Features
+   - Branch: `major` - Label:
+   ![](https://via.placeholder.com/15/4d3319/000000?text=+) **[branch]
+   major** - Content: Any change modifying a function signature in a
+   library class. Features require extensive LORIS-wide testing. New
+   complex systems and features spanning across multiple modules and
+   libraries. Deprecated functions clean-up.
+
+For more information about making well-organized pull requests,
+please read our in-depth Wiki page, ["Contributing to the
+Code"](https://github.com/aces/Loris/wiki/Contributing-to-the-Code).
+
+## Pull Request Title and Description
+
+To make it easier for reiewers to locate pull requests with wich they have
+expertise, we request that new pull requests follow a few conventions.
+
+#### Title
+
+The title should begin with square brackets enclosing the name of the
+module that you are changing followed by a brief description.
+
+e.g. This is a well-formed title.  > [Imaging Uploader / Server Process
+Manager] Fix invalid upload state
+
+If you are instead editing the `Core` libraries or the `tools` directory,
+etc., you can supply these values in place of a module name.
+
+There should be plenty of other examples in the list of pull requests
+in the main code repository.
+
+#### Description
+
+When opening a pull request on GitHub you will see a pull request
+template. Please fill out each heading with detailed information on your
+code changes, suggested testing instructions, and links to open GitHub
+issues and/or Redmine tickets (if applicable).
+
+## Some Things To Keep In Mind
+
+* If your changes require any table modifications:
+    1. Review our [SQL standard](./docs/SQLModelingStandard.md).
+    2. Modify the `SQL/0000*.sql` file(s) with your changes. These patches
+    are applied during the LORIS install.  3. Include a patch to be
```

Comment:
3. and 4. should be at column as 1. and 2. for rendering.

Reply:
@PapillonMcGill This is actually how it's intended to appear. 1. and 2. are sub instructions underneath the first `*`.